Tests/ClientTests/TabCellTests.swift

```diff
@@ -7,8 +7,46 @@
 import XCTest
 
 class TabCellTests: XCTestCase {
+    private func testState1() -> TabCellState {
```

Comment:
In the projectwe use a createSubject function precisely like this and add it at the bottom of the class

Reply:
Just glancing at our existing tests it looks like something like `createMockState1()` might be more appropriate? These tests will probably have more than one `state` to inject eventually, and it looks like `createSubject` is usually used to create the instance (`TabCell`) being tested? Happy to rename to whatever naming scheme is preferred though.

Reply:
createState makes sense I will just use instead of 1 a meaningful name life createDefaultState depending on the intention in a couple of months it will not be evident what createState1 meant for the test